skrub/_tabular_learner.py

```diff
@@ -48,19 +48,25 @@ def tabular_learner(estimator, n_jobs=None):
       missing values itself.
     - the last step is the provided ``estimator``.
 
-    **Note:** ``tabular_learner`` is a recent addition and the heuristics used
-    to define an appropriate preprocessing based on the ``estimator`` may change
-    in future releases.
+    Read more in the :ref:`User Guide <table_vectorizer>`.
+
+    .. note::
```

Comment:
:) I had done this at first but I thought the way notes are rendered with our template gives them too much visual importance for this comment